posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,584 @@
+
+|https://jakarta.ee/specifications/bean-validation/3.0/[Bean Validation 3.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/beanValidation-3.0.html[beanValidation-3.0]
+
+|https://jakarta.ee/specifications/debugging/2.0/[Debugging Support for Other Languages 2.0]
+|No change
+|Not applicable
+
+|https://jakarta.ee/specifications/enterprise-beans/4.0/[Enterprise Beans Lite 4.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/enterpriseBeansLite-4.0.html[enterpriseBeansLite-4.0]
+
+|https://jakarta.ee/specifications/managedbeans/2.0/[Managed Beans 2.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/managedBeans-2.0.html[managedBeans-2.0]
+
+|https://jakarta.ee/specifications/transactions/2.0/[Transactions 2.0]
+|No change
+|Not applicable (link:{url-prefix}/docs/latest/reference/javadoc/liberty-jakartaee10-javadoc.html?package=allclasses-frame.html&class=jakarta/transaction/package-summary.html[Javadoc])
+
+|===
+
+[#platform]
+== Jakarta EE Platform 10
+[cols="4,2,3"]
+|===
+|Specification |Updates |Liberty Feature Documentation
+
+|https://jakarta.ee/specifications/webprofile/10/[Jakarta EE Web Profile 10 Specification]
+|Major update
+|<<webprofile, See previous table>>
+
+|https://jakarta.ee/specifications/authorization/2.1/[Authorization 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/appAuthorization-2.1.html[appAuthorization-2.1]
+
+|https://jakarta.ee/specifications/activation/2.1/[Activation 2.1]
+|Minor update
+|Not applicable (link:{url-prefix}/docs/latest/reference/javadoc/liberty-jakartaee10-javadoc.html?package=allclasses-frame.html&class=jakarta/activation/package-summary.html[Javadoc])
+
+|https://jakarta.ee/specifications/batch/2.1/[Batch 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/batch-2.1.html[batch-2.1]
+
+|https://jakarta.ee/specifications/connectors/2.1/[Connectors 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/connectors-2.1.html[connectors-2.1]
+
+|https://jakarta.ee/specifications/mail/2.1/[Mail 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mail-2.1.html[mail-2.1]
+
+|https://jakarta.ee/specifications/messaging/3.1/[Messaging 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/messaging-3.1.html[messaging-3.1]
+
+|https://jakarta.ee/specifications/enterprise-beans/4.0/[Enterprise Beans 4.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/enterpriseBeans-4.0.html[enterpriseBeans-4.0]
+
+|===
+
+To enable all the Jakarta EE Platform 10 features, add the following to your `server.xml`:
+[source,xml]
+----
+  <featureManager>
+    <feature>jakartaee-10.0</feature>
+  </featureManager>
+----
+
+Similarly, to enable all the Jakarta EE Web Profile 10 features, add instead:
+[source,xml]
+----
+  <featureManager>
+    <feature>webProfile-10.0</feature>
+  </featureManager>
+----
+

```

Comment:
Should add the snippet on how to enable all the Jakarta EE Core Profile 10 feature.